qiskit_experiments/visualization/drawers/base_drawer.py

```diff
@@ -552,27 +554,34 @@ def image(
     def figure(self):
         """Return figure object handler to be saved in the database."""
 
-    def config(self) -> Dict:
+    def config(self) -> DrawerConfig:
```

Comment:
This is an interface change, do we have to follow some type of deprecation policy?